File: cos/cos.map/src/main/java/cos/map/Land.java
package cos.map;

import almazko.microjson.JsObject;
import almazko.microjson.Json;
import cos.map.parser.MapParser;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;

public class Land {

    public static Lands load(Path dir) throws IOException {
        var baseFile = dir.resolve("base1.json");
        var mapFile =  dir.resolve("untitled.json");
        var base = (JsObject) Json.parseObject(Files.readString(baseFile));
        var map = Json.parseObject(Files.readString(mapFile));
        return MapParser.parse(map, base);
    }
}

File: cos/cos.map/src/main/kotlin/cos/map/Land.kt
package cos.map

import cos.json.JsObject
import cos.json.Json2
import cos.json.Json3

object Land {

    @JvmStatic
    fun load(): Lands {

        val x =   Json3.parse("""{"trade": "https://login.primexbt.com/", "demo": "https://web.primexbt.com/", "api":"api.primexbt.com", "maintenance": true, "enable_recaptcha": true, "production": true, "xdStorageUrl": "https://primexbt.com/xd-storage"}""")

        val base = Json2.parse(javaClass.getResource("/base1.json").readText()) as JsObject
        val map = Json2.parse(javaClass.getResource("/map.json").readText()) as JsObject
        return MapParser.parse(map, base)
    }
}
